### ISSUE TITLE
where did we use the `call` function of YoloLayer?

### ISSUE THREAD

**Issue by jianing-sun** [experiencor/keras-yolo3#123]
Hi everyone, I must missed it - I didnt see when the code compute the yolov3 loss function which is under the `call` function of YoloLayer in `yolo.py`. I understannd `dummy loss` is just sth fake as we want to use `model.compile`. But where did we call the `call` function to compute the real yolov3 loss?

Thanks a lot!

**Comment by jianing-sun** [comment]
sorry I just figured it out - it is overwriting the `call` method from keras.engine.topology.Layer which will be triggered automatically.
I'm going to close this question. = )
